egui/src/widgets/plot/transform.rs

```diff
@@ -109,6 +129,16 @@ impl PlotBounds {
         self.translate_y(delta.y as f64);
     }
 
+    pub(crate) fn add_relative_margin_x(&mut self, margin_fraction: Vec2) {
+        let width = self.width().max(0.0);
+        self.expand_x(margin_fraction.x as f64 * width);
+    }
+
+    pub(crate) fn add_relative_margin_y(&mut self, margin_fraction: Vec2) {
+        let height = self.height().max(0.0);
+        self.expand_y(margin_fraction.y as f64 * height);
+    }
+
     pub(crate) fn add_relative_margin(&mut self, margin_fraction: Vec2) {
```

Comment:
let's remove this unused function to appease clippy

Reply:
done